src/boot/meson.build

```diff
@@ -344,6 +344,8 @@ if host_machine.cpu_family() in ['aarch64', 'arm', 'x86_64', 'x86']
         systemd_boot_sources += files('bcd.c')
 endif
 
+sources += libefi_sources + systemd_boot_sources + stub_sources + addon_sources
```

Comment:
This is going a wrong direction. We get rid of such runon lines e.g. spec files, even though that was the style 15 years ago, and with good reason.